clients/fides-js/src/lib/consent.tsx

```diff
@@ -58,7 +58,7 @@ export const initOverlay = async ({
             parentElem
           );
           debugLog(options.debug, "Fides overlay is now showing!");
-        } else if (experience.delivery_mechanism === DeliveryMechanism.LINK) {
+        } else {
```

Comment:
might want to double check this logic

Reply:
I'll update when I'm working on [link generation here](https://github.com/ethyca/fides/issues/3349)- we basically need to always bind link to open modal given that link element exists on page AND we have an overlay experience config.